third_party/blink/renderer/platform/wtf/hash_table.h

```diff
@@ -1103,6 +1129,55 @@ HashTable<Key, Value, Extractor, HashFunctions, Traits, KeyTraits, Allocator>::
       const_cast<const HashTable*>(this)->Lookup<HashTranslator>(key));
 }
 
+template <typename Key,
```

Comment:
Did you write this yourself?
Do you have a reference you can point to for this or can you explain it?